src/rust/ide/lib/view/src/graph_editor/builtin/visualization/native/bubble_chart.rs

```diff
@@ -2,10 +2,10 @@
 
 use crate::prelude::*;
 
-use crate::component::visualization::*;
-use crate::component::visualization;
-use crate::frp;
+use crate::graph_editor::component::visualization::*;
+use crate::graph_editor::component::visualization;
```

Comment:
not new but somewhat puzzling — why do we import both module and everything from the module?